src/StorageFlattening.cpp

```diff
@@ -245,6 +260,21 @@ class FlattenDimensions : public IRMutator {
             Expr store = Call::make(value.type(), Call::image_store,
                                     args, Call::Intrinsic);
             return Evaluate::make(store);
+        } else if (in_gpu && textures.count(op->name)) {
+            debug(2) << " lower texture store to " << op->name << "\n";
```

Comment:
debug(2) is too noisy for this. Probably can just cut them, or reduce to debug(4) if you want to keep them.